packages/Molecules/Form/fields/fieldRplselect.vue

```diff
@@ -1,118 +1,384 @@
 <template>
-  <multiselect
-    :options="options"
-    :value="localValue"
-    v-bind="selectOptions"
-    :placeholder="schema.placeholder"
-    :custom-label="customLabel"
-    :max="schema.max || null"
-    :disabled="disabled"
-    @input="updateSelected"
-    @search-change="onSearchChange"
-    @tag="addTag"
-    :option-height="selectOptions.optionHeight"
-  >
-  </multiselect>
+  <div class="rpl-select" :class="{'rpl-select--open' : isOpen}">
+    <div v-if="$breakpoint.s" class="rpl-select__native">
```

Comment:
Currently we have this field visually hidden at breakpoint "s" and above. I'm finding that there's now 2 ways to control the select box:
* by focusing on the `<select>` option below (hidden by screen but still works for screen readers)
OR
* by focusing on the rpl-select__trigger `<div>` below, which works for both screen readers and visually.

I think there should only be one input method to avoid confusion.

Also - using `$breakpoint.s` means any screen size below `s` (576 and under) will be false, meaning this will NOT show on mobile.

Reply:
yes, i think this was around the wrong way when you looked at it.